Automattic/wp-calypso
# Changing logo maker logo and copy to Fiverr.

We're testing Fiverr as the logo partner for our customers. This PR changes the logo and copy for the logo maker cards in Calypso. Those two cards are a small rectangle on My Home and a full card on the Marketing Tools page.

The updated card on My Home looks like:
![image](https://user-images.githubusercontent.com/35781181/97691297-94ce2b00-1a74-11eb-9836-18e7aba4cf48.png)

And the updated card on Marketing Tools looks like:
![image](https://user-images.githubusercontent.com/35781181/97691371-aadbeb80-1a74-11eb-9c30-88ff182a0d85.png)

The link is unchanged and still points to https://wp.me/logo-maker

The link on that page will be updated in a separate WPcom patch. Check pau2Xa-2bi-p2 for additional details.

#### Testing instructions

* Checkout this PR and start Calypso.
* Navigate to Calypso using any account.
* Click on My Home if you're not already there and check the Quick Links section on the right.
* Confirm that the correct copy and logo is there. Click the link and confirm that you're routed through the redirect (the redirect itself isn't updated yet).
* Navigate to the Marketing pages under Tools in the left nav.
* Confirm that the logo maker card shows the Fiverr logo and that the copy mentioned Fiverr.
* Click the link and confirm that you're routed through the redirect.
